CHANGELOG.md

```diff
@@ -39,6 +39,11 @@
 
   http.ListenAndServe(":8000", r)
   ```
+  
+### Known Issues and Workarounds
+
+* If a .NET agent is initiating traces as the root service, do not upgrade your downstream 
```

Comment:
Is it worth specifying that this is for **distributed** traces? (I assume customers would understand either way, but I feel like there are multiple contexts for the word "traces".)